client/src/pages/Home/Home.js

```diff
@@ -1,32 +1,182 @@
 import React from "react";
+
+import Mailchimp from "react-mailchimp-form";
+
 import Header from "../../components/Header";
 import Footer from "../../components/Footer";
+import styled from "@emotion/styled";
+import PictureTest from "../../assets/images/picture-test-1.jpg";
+import PictureTest2 from "../../assets/images/picture-test-2.jpg";
+import PictureTest3 from "../../assets/images/picture-test-3.jpg";
+import instagramSrc from "../../assets/icons/social-instagram.svg";
```

Comment:
You are using different naming conventions

Reply:
True!